sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_models.py

```diff
@@ -7,21 +7,29 @@
 from typing import List, Any
 from six import with_metaclass
 from enum import Enum
+from azure.core.exceptions import HttpResponseError
 from azure.core import CaseInsensitiveEnumMeta
 
 
 class UploadLogsStatus(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     SUCCESS = "Success"
     PARTIAL_FAILURE = "PartialFailure"
+    FAILURE = "Failure"
 
+class UploadLogsError:
+    """The errors for any failed logs
+    """
+    def __init__(self, **kwargs) -> None:
```

Comment:
Should this be:
```python
def __init__(self, *, failed_logs=None, error=None) -> None:
```

Reply:
These are ivars - I don't see why we need to make them positional